File: Baekjoon/18406_럭키 스트레이트/B_18406.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim().split('').map(v => parseInt(v));
const mid = input.length / 2;

let leftSums = 0;
let rightSums = 0;

for (let i = 0; i < mid; i++) {
    leftSums += input[i];
}


for (let i = mid; i < input.length; i++) {
    rightSums += input[i];
}

leftSums === rightSums ? console.log('LUCKY') : console.log('READY');

File: baekjoon/18406_럭키 스트레이트/B_18406.py
def get_sums(arr, start_idx, last_idx):
    sums = 0
    for i in range(start_idx, last_idx):
        sums += arr[i]

    return sums


N = list(map(int, input()))
length = len(N)
half = length // 2

left_sums = get_sums(N, 0, half)
right_sums = get_sums(N, half, length)

if left_sums == right_sums:
    print('LUCKY')
else:
    print('READY')

# 이것이 코딩테스트다 수록 문제
